src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/oracle/OracleEngineSchemaTest.java

```diff
@@ -17,19 +17,25 @@
 
```

Comment:
I think all these tests could all be in AbstractEngineSchemaTest, since they are not Oracle-specific.